document/core/index.rst

```diff
@@ -15,6 +15,7 @@ WebAssembly Specification
    binary/index
    text/index
    appendix/index
+   changes
```

Comment:
AIUI, this is a requirement for all documents, though it doesn't have to be in this format. Looking at other technical reports, they generally have a fairly fine-grained log of changes. See for example https://www.w3.org/TR/2018/REC-WOFF2-20180301/#changes or https://www.w3.org/TR/2018/REC-css-ui-3-20180621/#changes.

It was easier for me to just write up each individual change without judging whether it was substantive or not, and how to group it, but I think we can condense this down.

Reply:
Yes, but as far as I understand the W3C guidelines [here](
https://www.w3.org/2018/Process-20180201/#working-draft), the log of changes between draft versions isn't intended to be part of the document itself, but rather described as meta documentation on the process level. I think it still makes sense to link it from the web page as you did for the core spec, but I wanted to clarify its status as part or not part of the doc itself. For the JS & Web spec it might be better to put the logs on separate pages as well, in order to clarify the distinction between the docs that we want to standardise and meta information about those docs.

Probably no need to optimise the presentation of the logs for this meta purpose.

Reply:
OK, I see what you mean. I've moved the changes for the other documents to their own files to make it clearer that they are not part of the core document.

Reply:
LGTM